openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<p>Not seeing any output on my end on any of the cells in this notebook due to them being distributed. Not sure if this is a "my system" problem or not as far as when they get built by GH and sent to the server for display</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I've posted some instructions for setting up mpi. See if those work for you.

Reply:
That worked